packages/core/src/auto.ts

```diff
@@ -1210,15 +1210,6 @@ export default class Auto {
       await gitReset();
     }
 
-    let latestTag: string;
-
-    try {
-      latestTag = await this.git.getLatestTagInBranch();
-    } catch (error) {
-      latestTag = await this.git.getFirstCommit();
-    }
-
-    const commitsInRelease = await this.release.getCommits(latestTag);
```

Comment:
Removed this because getting all the commits in a canary release with no recent release can lead to hitting the rate limits. With canaries this is especially easy to hit because they are made so often.